### PR TITLE
moved style tag rule into infragram.css

### PR DESCRIPTION
**Pull request by on2onyekachi**
moved the style rule from index.html into infragram.cs
Screenshot 2022-03-30 at 10 30 46
s

### CODE REVIEW

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Comment by on2onyekachi** [comment]
oh okay. Thanks, I would do just that.

File: dist/infragram.css [hunk 1]
```diff
@@ -97,3 +97,9 @@ ul.dropdown-menu li:hover{
   cursor: pointer;
   background-color: #e0e0e0;
 }
+#file-sel {
+  position:absolute; left:0;
```
[the remark this answers is not in the document]
Comment (on2onyekachi) [reply dist/infragram.css:101]: Oh okay. Thanks. I would do just that.

**Review by TildaDares — APPROVED** [review]
Great work @on2onyekachi. Thank you!

**Comment by TildaDares** [comment]
Congratulations 🎉

**Pull request merged**